pkg/domain/infra/abi/play.go

```diff
@@ -732,6 +732,16 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		if err != nil {
 			return nil, nil, err
 		}
+
+		// Make sure to complete the spec (#17016)
+		warn, err := generate.CompleteSpec(ctx, ic.Libpod, specGen)
+		if err != nil {
+			return nil, nil, err
+		}
+		for _, w := range warn {
+			fmt.Fprintf(os.Stderr, "%s\n", w)
+		}
```

Comment:
I see the other callers do the same but shouldn't we use `logrus.Warn()` for this? It feels very ugly to just print somewhere in the backend where the callers have no control over it. I would argue that podman --log-level=error should not show such messages if they are just warnings.

Reply:
I also don't like printing to stderr.  But as you mentioned, it's consistent with other call sites, so I didn't change it.

I am supportive of changing those to `logrus.Warn*`.  There's probably more such cases across the code base.